test/e2e/framework/framework.go

```diff
@@ -149,7 +149,9 @@ func (f *Framework) DeleteEtcdOperatorCompletely() error {
 	if err != nil {
 		return err
 	}
-	err = retryutil.Retry(2*time.Second, 5, func() (bool, error) {
+	// On k8s 1.6.1, grace period isn't accurate. It took ~10s for operator pod to completely disappear.
```

Comment:
Right. This is an issue we encountered in 1.6 beta releases. We thought it would be fixed in stable release. We should report to upstream and revisit this again.